### PR TITLE
Fland mailroom garage door link

### LINKED ISSUE #31145: MAP_FLAND: Mailroom Garage Door Opener isn't actually linked to the doors
image

### PR DESCRIPTION
**Pull request by thetolbean**
## About the PR
Connect the remote signaler to the mailroom garage doors 

## Why / Balance
Fixes #31145 

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by thetolbean** [comment]
Test fail is not related to mapping change

File: Resources/Maps/fland.yml [hunk 1]
```diff
@@ -180555,11 +180555,15 @@ entities:
       parent: 13329
   - uid: 11854
     components:
-    - type: MetaData
-      name: Mailroom Garage Door Opener
     - type: Transform
-      pos: 21.53215,-13.842299
+      pos: 21.52355,-14.18487
```
Comment (Brandon-Huu) [inline Resources/Maps/fland.yml:180559]:
```suggestion
      pos: 21.53215,-13.842299
```

File: Resources/Maps/fland.yml [hunk 2]
```diff
@@ -180555,11 +180555,15 @@ entities:
       parent: 13329
   - uid: 11854
     components:
-    - type: MetaData
-      name: Mailroom Garage Door Opener
```
Comment (Brandon-Huu) [inline Resources/Maps/fland.yml:180559]: Don't remove this

**Review by Brandon-Huu — CHANGES_REQUESTED** [review]

File: Resources/Maps/fland.yml (hunk 2 above)
Comment (thetolbean) [reply Resources/Maps/fland.yml]: Resolved with most recent commit

File: Resources/Maps/fland.yml (hunk 1 above)
Comment (thetolbean) [reply Resources/Maps/fland.yml:180559]: Remote is still on same initial table, just with slightly different positioning.

**Pull request merged**